docs/configuration/pepr-policies.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Pepr Policies
+type: docs
+weight: 3
+---
+
+## Common Pepr Policies for UDS Core
+
+These policies are based on the [Kyverno](https://kyverno.io/policies/) policies.
+
+Exemptions can be specified by a [UDS Exemption CR](https://github.com/defenseunicorns/uds-core/blob/v0.27.0/src/pepr/operator/README.md). These take the place of Kyverno Exceptions.
```

Comment:
@MxNxPx noting this as a place we can link the exemption docs when added instead of the operator readme.